api/server/router/system/system_routes.go

```diff
@@ -21,6 +22,16 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+type invalidRequestError struct {
```

Comment:
Actually wondering why we added this one in the first place, instead of just using `errdefs.InvalidParameter()` utility (which should give the same).

Anyway, that's something to look at separately I guess. (Perhaps from that perspective, keep the first commit out of this PR)